### DISCUSSION TITLE
Bad appearance of release 13

### DISCUSSION THREAD

**Discussion by pepo-ec** [apache/netbeans#3746] [General]
I installed version 13 of NetBeans but when I run it looks pretty bad, instead, if I execute the version 12 everything looks good.  How do I fix this problem?

**Comment by pepo-ec** [comment]
I've corrected:

Tools> Options> Appearance> Look and Feel

Election: Metal

**Comment by asbachb** [comment]
Something looking "pretty bad" is not very specific. How should anybody know what you're talking about? Please post a screenshot.
